src/ArmTemplates/Common/Templates/TemplateBuilder.cs

```diff
@@ -0,0 +1,108 @@
+﻿// --------------------------------------------------------------------------
+//  <copyright file="TemplateBuilder.cs" company="Microsoft">
+//      Copyright (c) Microsoft Corporation. All rights reserved.
+//  </copyright>
+// --------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Constants;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Templates.Abstractions;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Extractor.Models;
+
+namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Templates
+{
+    static class TemplateBuilder
```

Comment:
Again, not a fan of making everything static. We can't do proper unit tests with static stuff.

Reply:
answered [here](https://github.com/Azure/azure-api-management-devops-resource-kit/pull/639#discussion_r823696751)